doc/sql_guide.md

```diff
@@ -0,0 +1,291 @@
+# Test Objective
+Structured Query Language (SQL), SQL is an ANSI (American National Standards Institute) standard computer language. Its functions include data query, data manipulation, data definition and data control. It is a general relational database language with strong functions. It is now the standard language for relational databases. (such as SELECT, UPDATE, DELETE, INSERT, WHERE and etc)   
+CUBRID SQL test, SQL detection for each build to detect the basic functions of the database and ensure the correct output of commands.
+  
+# General SQL Test  
+## Test deployment  
+* Install CTP: [CTP_install_guide](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md)  
```

Comment:
Change to:

Install CTP
Please follow guides to [install CTP on Linux platform](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#1-install-ctp-in-linux-platform) or [install CTP on Windows platform](...).